resources/views/backend/posts/show.blade.php

```diff
@@ -25,127 +30,92 @@
+                    @break
+                @endswitch
             </ul>
+            <!-- /.tab-content -->
+
             <div class="tab-content">
-              <div class="active tab-pane" id="activity">
-                <!-- Post -->
-                <div class="post">
-                  <div>
-                        <span class="username">
-                         <a href="#" class="h2">Jonathan Burke Jr.</a></br>
-                        </span>
-                        <ol class="breadcrumb">
-                          <li><b>{{ __('post.by') }} </b><i>User </i></li>
-                          <li><b>{{ __('post.score') }} :</b><i>9</i></li>
-                          <li><b>{{ __('post.date') }} :</b><i> 20.10.2017</i></li>
-                          <li><i class="fa fa-trash-o text-danger"></i></li>
-                        </ol>
-                  </div>
-                  <!-- /.user-block -->
-                  <p>
-                    Lorem ipsum represents a long-held tradition for designers,
-                    typographers and the like. Some people hate it and argue for
-                    its demise, but others ignore the hate as they create awesome
-                    tools to help create filler text for everyone from bacon lovers
-                    to Charlie Sheen fans.
-                  </p>
-                </div>
-                <!-- /.post -->
-              </div>
-              <!-- /.tab-pane -->
-              <div class="tab-pane" id="timeline">
-                  <!-- Post -->
-                  <div class="post">
-                    <div>
-                          <span class="username">
-                           <a href="#">Jonathan Burke Jr.</a></br>
-                          </span>
-                          <ol class="breadcrumb">
-                            <li><b>{{ __('post.by') }} </b><i>User </i></li>
-                            <li><b>{{ __('post.score') }} :</b><i>9</i></li>
-                            <li><b>{{ __('post.date') }} :</b><i> 20.10.2017</i></li>
-                            <li><i class="fa fa-trash-o text-danger"></i></li>
-                          </ol>
+                <!-- /.display review -->
+
+                    <div class="active tab-pane" id="activity">
+                        <!-- Post -->
+                        <div class="post">
+                            <div>
+                                <div class="h2">
+                                    {{$post->name}}
+                                </div>
+                                <ol class="breadcrumb">
+                                    @if ($post->type == 1)
+                                        <li><b>{{ __('post.score') }} :</b>
+                                            <i>
+                                                {{$post->rating}}
+                                            </i>
+                                        </li>
+                                    @endif
```

Comment:
dùng Attribute luôn nha anh

Reply:
attribute la sao nghia. co the giai thich ko??

Reply:
mình tạo ra 1 Attribute trong model Post để xử lí  sau đó ngoài view mình chỉ show ra thôi